platform_versioned_docs/version-24.1/cloud/changelog.mdx

```diff
@@ -5,41 +5,38 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
-### v24.2.0_cycle23 Cloud - 2 September 2024
-
-- Fix: [Data Explorer] enable task working directory navigation using Data Explorer in personal workspaces by adding routing
-- Fix: Remove -/+ increment buttons from numeric input components in Platform, including compute environment and launch form interfaces
-- Fix: Hide compute environment variables section actions when component is disabled (previously indicated that environment variables could be edited in this state)
-
-### 24.2.0_cycle21 Cloud - 13 August 2024
-
-- Feature: Add global Nextflow configuration support in compute environments
-- Feature: Add flexibility for pipeline names in workspaces
-- Feature: Add tag propagation to launch templates
-- Feature: Add managed identities support for manual Azure Batch compute environments
-- Feature: Implement custom launch container logic
-- Fix: Improve workflow launch screen look and feel
-- Fix: Allow pipeline work directory to be changed during pipeline launch
-- Fix: Handle special characters in prompt modal confirmation text regex
+### 24.2.0 - 2 September 2024
```

Comment:
The `_cycle` for each Cloud release is like a patch or minor release identifier, making each release unique. It's better to keep this than to have all releases in the major cycle have the same heading.